cocos/tween/actions/action-interval.ts

```diff
@@ -493,17 +497,20 @@ export class Repeat extends ActionInterval {
     private _actionInstant = false;
     private _innerAction: FiniteTimeAction | null = null;
 
-    constructor (action?: any, times?: any) {
+    constructor (action?: FiniteTimeAction, times?: number) {
         super();
-        times !== undefined && this.initWithAction(action as FiniteTimeAction, times as number);
+        this.initWithAction(action, times);
     }
 
     /*
      * @param {FiniteTimeAction} action
      * @param {Number} times
      * @return {Boolean}
      */
-    initWithAction (action: FiniteTimeAction, times: number): boolean {
+    initWithAction (action?: FiniteTimeAction, times?: number): boolean {
+        if (!action || times === undefined) {
+            return false;
+        }
```

Comment:
The function declaration says `action` and `times` are optional, but it will return false if not passing them. It is conflict.

Reply:
I dont' know what it conflicts with.

In fact, lots of `initWithXXX` are invoked in `constructor` and its return value is ignore in construtor.

Reply:
I just discuss the API design here. If the parameters are optional, then not passing the parameters should be valid. But the implementation doesn't allow it.